### ISSUE TITLE
How to port the function of Mat::zeros?

### ISSUE THREAD

**Issue by higoalespn** [hybridgroup/gocv#244]
## Description

the coder as follow:

Mat Zeros(int rows, int cols, int type ) {
    Mat zero;
    zero = cv::Mat::zeros(rows, cols, type);
    return zero;
}

but have some problem:
src\gocv.io\x\gocv\core.cpp: In function 'cv::Mat* Zeros(int, int, int)':
src\gocv.io\x\gocv\core.cpp:584:43: error: cannot convert 'cv::MatExpr' to 'Mat {aka cv::Mat*}' in assignment
     zero = cv::Mat::zeros(rows, cols, type);
I have added the code for problem:

cv::Mat& cv::Mat::operator=(const cv::MatExpr &expr);

but have new problem:
src\gocv.io\x\gocv\core.cpp:580:52: error: declaration of 'cv::Mat& cv::Mat::operator=(const cv::MatExpr&)' outside of class is not definition [-fpermissive]
 cv::Mat& cv::Mat::operator=(const cv::MatExpr &expr);
                                                    ^
src\gocv.io\x\gocv\core.cpp: In function 'cv::Mat* Zeros(int, int, int)':
src\gocv.io\x\gocv\core.cpp:584:43: error: cannot convert 'cv::MatExpr' to 'Mat {aka cv::Mat*}' in assignment
     zero = cv::Mat::zeros(rows, cols, type);

## Your Environment

* Operating System and version:Windows10
* OpenCV version used:3.4.1
* How did you install OpenCV?
* GoCV version used:0.13.0
* Go version:go version go1.8 windows/amd64
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?

**Comment by deadprogram** [comment]
Hello @higoalespn looks to me like you might need to define your function something like this:

```
Mat Zeros(int rows, int cols, int type ) {
   return new cv::Mat(cv::Mat::zeros(rows, cols, type));
}
```

**Comment by deadprogram** [comment]
This was released with v0.26.0 so now closing. Thank you everyone!
